lms/static/coffee/src/instructor_dashboard/student_admin.coffee

```diff
@@ -223,6 +224,30 @@ class @StudentAdmin
           full_error_message = interpolate_text(error_message, {student_id: unique_student_identifier})
           @$request_response_error_ee.text full_error_message
 
+  # Mark a student to skip entrance exam
+    @$btn_skip_entrance_exam.click =>
+      unique_student_identifier = @$field_entrance_exam_student_select_grade.val()
+      if not unique_student_identifier
+        return @$request_response_error_ee.text gettext("Enter a student's username or email address.")
+      confirm_message = gettext("Do you want to allow this student ('{student_id}') to skip the entrance exam?")
+      full_confirm_message = interpolate_text(confirm_message, {student_id: unique_student_identifier})
+      if window.confirm full_confirm_message
+        send_data =
+          unique_student_identifier: unique_student_identifier
+
+        $.ajax
+          dataType: 'json'
+          url: @$btn_skip_entrance_exam.data 'endpoint'
+          data: send_data
+          type: 'POST'
+          success: @clear_errors_then ->
+            success_message = gettext("This student ('{student_id}') will skip the entrance exam.")
```

Comment:
If I'm reading this correctly, this is the message that will be returned by the server in this situation. Why doesn't the code just show the message, rather than creating its own message? Alternatively, why is the server's message localized if it isn't ever shown to a user?